class DomainError(Exception):
    pass


class NotFoundError(Exception):
    pass
